README.md

```diff
@@ -1,8 +1,38 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
+# Adrian Zaia . Jarvis Consulting
+I am a Computer Science Graduate from Ryerson University. I first started learning about programming in High School with Java and HTML, i gained an interest in seeing programs come to life through my own thinking and implementation, it felt like learning a new langauge to speak. I pursued with this passion for programming and attended Ryerson University for my degree and landed a co-op job in the summer with RBC as an Operational Risk Analyst.
+
+
+# Skills
+
+Proficient: Microsoft Suite, Java, HTML, SQL, CSS
+
+Competent: C/C++, Python, C#, Javascript, Adobe Suite
+
+Familiar: Unity Game Engine, Unreal Game Engine, Github, Docker, Node.JS
+
+# Jarvis Projects
+
+Linux Cluster Monitor [Link](https://github.com/jarviscanada/jarvis_data_eng_AdrianZaia/tree/master/linux_sql): Developed tools that can monitor nodes that are connected into a Linux Cluster, the tools allow users to track hardware specifications and reource usage in real time and registering the collected data into a database. This program was created using tools such as, Linux Instance, Git, GitHub, Docker, PSQL and Crontab.
```

Comment:
Linux Cluster Monitor [Link](https://github.com/jarviscanada/jarvis_data_eng_AdrianZaia/tree/master/linux_sql): Developed tools that can monitor nodes that are connected into a Linux Cluster, the tools allow users to track hardware specifications and resource usage in real-time and persist the collected data into a PostgreSQL database. Data analytics and reproting SQL .... Docker, crontab, testing, GIt...